infrastructure/templates/template.json

```diff
@@ -1361,7 +1361,25 @@
     "searchReaderManagedIdentityName": "[concat(parameters('subscription'), '-', parameters('environment'), '-id-srch-reader')]",
     "searchServiceApiVersion": "2025-02-01-preview",
     "userAssignedIdentitiesApiVersion": "2024-11-30",
-    "storageAccountBackupContributorRoleId": "[subscriptionResourceId('Microsoft.Authorization/roleDefinitions', 'e5e2a7ff-d759-4cd2-bb51-3152d37e2eb1')]"
+    "storageAccountBackupContributorRoleId": "[subscriptionResourceId('Microsoft.Authorization/roleDefinitions', 'e5e2a7ff-d759-4cd2-bb51-3152d37e2eb1')]",
+    "baseAdminAllowedOrigins": [
+      "[concat('https://', parameters('adminUri'))]"
+    ],
+    "devAdminAllowedOrigins": [
+      "https://localhost:5021",
+      "http://localhost:5021"
+    ],
+    "conditionalDevAdminAllowedOrigins": "[if(equals(parameters('environmentName'), 'Development'), variables('devAdminAllowedOrigins'), createArray())]",
+    "basePublicAllowedOrigins": [
+      "[concat('https://', parameters('domain'))]",
+      "[concat('https://', variables('publicAppName'), '.azurewebsites.net')]"
+    ],
+    "devPublicAllowedOrigins": [
+      "https://localhost:3000",
+      "http://localhost:3000",
+      "https://ees-public-site-dkbjhuezemehdzff.a03.azurefd.net"
```

Comment:
You could separate out the changes related to Azure Front Door to a separate PR, if that's bit more experimental at this point.

Or just add these changes to the branch you are working on to implement the main Azure Front Door changes.